src/ftest.jl

```diff
@@ -0,0 +1,75 @@
+type FTestResult
+    mod1::RegressionModel
```

Comment:
Should probably not store the original models, that can take a lot of space and can be tracked separately if needed.